### ISSUE TITLE
Custom located Controls are offset if using a different DPI-setting in the display-settings

### ISSUE THREAD

**Issue by xenocidic** [cadon/ARKStatsExtractor#151]
![arksbdisplayerror](https://cloud.githubusercontent.com/assets/25232432/22130044/cecfddd6-de77-11e6-824c-49a5fcff6953.png)

There is another visual error, on the Extractor page, it says "The TE can differ 0.1% due to" [sentence ends].

**Comment by cadon** [comment]
Hi, thanks for the report!

Can you describe how to reproduce that first issue? I can't see this. Do you have the most uptodate version?

I'll fix the second one.

**Comment by xenocidic** [comment]
I don't know how to reproduce it, it has always been like this. Would uploading my database help?

**Comment by cadon** [comment]
I don't think it's related to the creatures in the database. What version of windows do you use? Maybe it's related to that.

**Comment by xenocidic** [comment]
Windows 7 Professional

**Comment by cadon** [comment]
Hm, it could be related to scrolling, that's the only variable that could change the hardcoded position of the right creature-column. Can you try to resize the window of the tool, so scrollbars appear at the bottom, slide them a bit, enlarge a bit and click again on a species? That sounds a bit silly, but I've no idea what could cause this.

**Comment by xenocidic** [comment]
With the new version it looks a little different, but the scrolly trick didn't fix anything.

**Comment by Darkblood18** [comment]
I have a very similar issue in Windows 10, and also another problem where sometimes a button or other element will be out of the range of the window (the button that "Add new to library" in the extractor window is an example).

I seems that changing the resolution moves stuff around a bit, the problem with the "Add new to library" is solved if I increase my definition to 1920 x 1080. I usually use 1600 x 900 which is more comfortable in my small notebook screen. But the pedigree screen still looks all messy.

**Comment by cadon** [comment]
Hi, can you post a screenshot of the pedigree screen? The messier the better to find the error.

**Comment by Darkblood18** [comment]
Thanks for the answer. This screenshot has it all:

![capturar](https://cloud.githubusercontent.com/assets/25591231/22662447/39e42a5c-ec90-11e6-8cb8-d40d93a03916.PNG)

The Pedrigree is for "BFD16 (TF210) Precursor", his stats are not bellow the stat headings, and both his parents are obscured by his box (all arrows and graphical elements are also out of place, notice the green and red arrows besides the "parenting" grey arrows). Also, the descendants names are cropped (names have too little space in general).

**Comment by cadon** [comment]
Do you have by any chance changed the dpi-setting in windows? Maybe that changes how the controls are drawn. I.e. do you have very high resolutions on your displays?

**Comment by Darkblood18** [comment]
Yep. You nailed it. My laptop keeps complaining if I keep the resolution below 1920 x 1080 (ideal resolution and some such) but the screen is not big and I am too old to see the tiny characters at that definition. So I had windows to scale everything up by 125%. When I bring it back to 100% everything falls into place. Thanks!

Funny fact: windows has a "compatibility setting" that overrides DPI settings to a particular program. But it doesn't work. As far as I could see everything is scaled despite of this setting.

**Comment by cadon** [comment]
Great. I'll check if there's a way to set a scaling factor in the tool dependant on the DPI-setting.

**Comment by cadon** [comment]
Another user described that this issue got solved when they changed the dpi-settings and then changed it back. Can you try if that works for you as well? [Here's what they did in detail](https://www.reddit.com/r/playark/comments/5mgxfy/smart_breeding_now_can_extract_imprinted_creatures/ddhw6ih/)

**Comment by xenocidic** [comment]
On Windows 7, I right click desktop -> Screen resolution -> Make text and other items larger or smaller -> Set to 100% or 150% and Ark Smart Breeding will display properly. However, 125% is the Goldilocks setting for my 55" television and even when switching back and forth between 100%/150% and 125% applying and logging off after each setting does Ark Smart Breeding "snap back" as described by /u/Dramier. Perhaps this workaround does not work for Windows 7?

**Comment by VolatilePulse** [comment]
This issue is related to how Windows handles scaling. While this feature is used by many users, we are unaware of any potential fixes and are open to suggestions or recommendations for support in Window Forms applications.

This issue will remain open and be referenced from all future issues relating to it.
